src/tabs/test/tabs.spec.js

```diff
@@ -150,6 +150,41 @@ describe('tabs', function() {
     });
   });
 
+  describe('without active binding and index attributes', function() {
+    beforeEach(inject(function($compile, $rootScope) {
+      scope = $rootScope.$new();
+      scope.first = '1';
+      scope.second = '2';
+      elm = $compile([
+        '<uib-tabset>',
+        '  <uib-tab heading="First Tab {{first}}">',
+        '    first content is {{first}}',
+        '  </uib-tab>',
+        '  <uib-tab heading="Second Tab {{second}}">',
+        '    second content is {{second}}',
+        '  </uib-tab>',
+        '</uib-tabset>'
+      ].join('\n'))(scope);
+      scope.$apply();
+      return elm;
+    }));
+
+    it('should bind tabs content and set first tab active', function() {
+      expectContents(['first content is 1', 'second content is 2']);
+      expect(titles().eq(0)).toHaveClass('active');
+      expect(titles().eq(1)).not.toHaveClass('active');
+      expect(scope.active).toBe(1);
```

Comment:
DOH!  Sorry I missed that, I'm surprised it didn't fail when running the test in Chrome?